src/styles.scss

```diff
@@ -14,3 +16,61 @@ body {
   @apply bg-grey-100;
   overflow-y: scroll; /* Always show scrollbars to avoid page skipping about */
 }
+
+:root {
+  --font-family: theme("fontFamily.body");
+}
+
+// For now I'm just overriding some defaults here
+// Instead of extending the whole theme
+// Quick and dirty just to get some basic styling
```

Comment:
This is not how theming with PrimeNG is supposed to be done, but I couldn't be bothered to import a whole theme and customise it (yet). Not for the proof of concept at least.